README.md

```diff
@@ -16,14 +16,18 @@ The assumption is that if you are applying to software companies in Japan you wi
 
 | [Amazon Japan](https://www.amazon.jobs/en/locations/tokyo-area-japan?base_query=&job_count=10&result_limit=10&sort=relevant&location%5B%5D=tokyo-area-japan&cache) | Shinagawa-ku, Tokyo | Multiple sectors
 
-| [Abeja](https://abeja.asia) | Minato-ku, Tokyo | Data Analytics
```

Comment:
Not removed I just put it down on the startups list since its more of a startup than an established company with lots of revenue. I believe they only have something like 20-40 employees.